restler-core/src/main/java/net/researchgate/restdsl/results/EntityResult.java

```diff
@@ -17,12 +18,17 @@
  * General container for entity results
  * TODO: introduce redirects in the right manner
  */
-
+@Schema(description = "The response to to a restler query. Depending on the query, exactly one of the sub fields 'list' or 'multimap' is populated")
 @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EntityResult<T> implements EntityContainer<T> {
+    @Schema(description = "If the client query contains no groupBy parameter, then this field is populated")
     private EntityList<T> list;
+
+    @Schema(hidden = true, deprecated = true, description = "This field is never populated in restler")
     private EntityMap<T> map;
+
+    @Schema(description = "If the client query contains a groupBy parameter, then this field is populated", example = "null")
```

Comment:
example = "null" is not very helpful I'd say, I suggest to use a "real" one or remove it

Reply:
this is intende,  i was not able to express `exactly one of the fields is set` so i 
- left the list as default, 
- hid the map (it should actually be removed, because its never set
- nulled the multimap so that the swagger ui looks better